### PR TITLE
dapper

### PR DESCRIPTION
**Pull request by VigneshNatarajan27**
Hi @MaithiliyK . 

Please review and merge these topic in connecting to database topic. We have added dapper and SQLIte samples here. 

Regards,
Vignesh Natarajan

### CODE REVIEW

File: Binding Dapper using UrlAdaptor/Blazor WASM App/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Blazor DataGrid Microsoft SQL Server connectivity using UrlAdaptor
+
+A project that enables data binding and CRUD action handling in the Syncfusion Blazor DataGrid to a Microsoft SQL Server using UrlAdaptor feature.
+
+## Steps to run the sample
+
+1. Download or unzip the project
+
+2. This project consist of two applications so open and run both MyWebService and GridWASM_MSSQL_UrlAdaptor projects in Visual Studio 2022
```
Comment (MaithiliyK) [inline Binding Dapper using UrlAdaptor/Blazor WASM App/README.md]: project names are wrong. Correct the readme file in all the topics

**Review by MaithiliyK — CHANGES_REQUESTED** [review]
address my review comments

File: Binding Dapper using UrlAdaptor/Blazor WASM App/README.md (hunk 1 above)
Comment (VigneshNatarajan27) [reply Binding Dapper using UrlAdaptor/Blazor WASM App/README.md]: addressed the comments by correcting the project name in all the readme files.

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**